src/main/charts/bamboo/values.yaml

```diff
@@ -608,6 +608,10 @@ bamboo:
     #
     periodSeconds: 10
 
+    # -- Number of seconds after which the probe times out
+    #
+    timeoutSeconds: 1
```

Comment:
So if the liveness probe does not get a response after `1` second, thats its cue to trigger a failure and reboot the pod?

Reply:
It will consider this attempt as failed and then try again after `periodSeconds` and until either `successTreshold` or `failureTreshold` is reached.